.github/workflows/security-scan.yml

```
@@ -0,0 +1,86 @@
+name: Security Scan
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  scan:
+    runs-on: ['self-hosted', 'linux', 'large']
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    steps:
+    - uses: actions/checkout@v3
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
```

We'll probably want to use v3 to avoid the node 12 warnings:

```suggestion
      uses: actions/setup-go@v3
```